phunware/maas-mapping-ios-sdk
# Feature/restore landmark routing

Changes:
-
* Revert commit that retrofitted the samples for the previous version of the Mapkit SDK.
* Add markdown file to explain Landmark Routing sample.
* Added link to landmark routing markdown file to main scenario explanation
* Fixed some typos in some documentation

To test:
- 
Make sure your local Mapkit working copy is pointed at the latest `dev` branch.
Apply the patch in [this zip file](https://github.com/phunware/maas-mapping-ios-sdk/files/3922959/qa_setup_12_4.patch.zip) on top of this branch to configure building and route POIs (you may need to update the local path to your mapkit installation), then run `pod update` before building. Please ensure that landmark info is displayed in the Turn by Turn - Landmarks sample.
